### PR TITLE
Add os_keystone_domain_facts module

### PR DESCRIPTION
**Pull request by rcarrillocruz**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME

os_keystone_domain_facts
##### ANSIBLE VERSION

```
ansible 2.1.0 (devel cca084c89d) last updated 2016/04/05 14:55:00 (GMT +000)
  lib/ansible/modules/core: (detached HEAD cf01087a30) last updated 2016/04/05 14:55:07 (GMT +000)
  lib/ansible/modules/extras: (os_domain_facts 8d74914a63) last updated 2016/04/06 09:55:59 (GMT +000)
  config file = 
  configured module search path = Default w/o overrides
```
##### SUMMARY

This module gathers one or more OpenStack domains facts

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @rcarrillocruz. This module is going into community review.

We notice that you are one of the maintainers (@emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/openstack/os_keystone_domain_facts.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+extends_documentation_fragment: openstack
+version_added: "2.1"
+author: "Ricardo Carrillo Cruz (@rcarrillocruz)"
+description:
+    - Retrieve facts about a one or more OpenStack domains
+requirements:
+    - "python >= 2.6"
+    - "shade"
+options:
+   domain:
```
Comment (Shrews) [inline cloud/openstack/os_keystone_domain_facts.py]: Does not match the parameter name.

Comment (rcarrillocruz) [inline cloud/openstack/os_keystone_domain_facts.py]: done, thanks

**Comment by rcarrillocruz** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @rcarrillocruz. To the current maintainers, @emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit' or 'needs_revision' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Pull request merged**